e2e/spec/404.spec.e2e.ts

```diff
@@ -1,6 +1,83 @@
-Feature('404');
+Feature('404 error handling');
 
-Scenario('should show 404 error page', (I) => {
+//region 404 page for non-existing ILC route
+Scenario('Renders (SSR) global 404 page for non-existing ILC route', (I) => {
     I.amOnPage('/nonexistent-path');
     I.waitForText('404 not found', 10, 'body > div#body');
 });
+Scenario('Renders (CSR) global 404 page for non-existing ILC route', (I, peoplePage) => {
+    const notFoundPageLink = '#navbar a[href="/nosuchpath"]';
+
+    I.amOnPage(peoplePage.peopleUrl);
+    I.waitForElement(notFoundPageLink, 30);
+    I.click(notFoundPageLink);
+    I.waitForText('404 not found', 10, 'body > div#body');
+});
+//endregion 404 page for non-existing ILC route
+
+//region 404 page for non-existing News app route
+Scenario('Renders (SSR) global 404 page for non-existing News app route', (I) => {
+    I.amOnPage('/news/nonExisting');
```

Comment:
I see you added URLs here is `e2e/spec/pages/news.ts`
So you can use `newsPage.url.nonExistingRoute` here

Reply:
👍